bundles/EcommerceFrameworkBundle/CartManager/AbstractCart.php

```diff
@@ -952,11 +952,11 @@ protected function validateVoucherTokenReservations()
     protected static function isValidCartItem(CartItemInterface $item)
     {
         $product = $item->getProduct();
-        if ($product instanceof CheckoutableInterface && !$product instanceof MockProduct) {
```

Comment:
I would still keep `CheckoutableInterface` in case somebody has overridden the getProduct() method.

Reply:
@dvesh3 Okay, I add it